src/components/EntityInfo.vue

```diff
@@ -286,9 +287,10 @@ export default class EntityInfo extends Mixins(CommonMixin, EnumMixin) {
 #entity-info {
   background: $BCgovInputBG;
 
-  &.hover {
-    background: $app-bg-lt-blue;
-  }
+  // TODO: Curious about this one. Is this a design requirement?
```

Comment:
Yes it is.

Reply:
It's to show users what company information would change. (Tracey's idea.)

Reply:
kk no prob, it does highlight the entire entity info area when you hover over the btn, so it seemed odd. I will put back in tho!

Reply:
I thought you were going to leave this in.

Reply:
Added back in